.changelog

```diff
@@ -4,8 +4,13 @@
 # - Fixed error in Bat section of Upgrade Guide (bsc#1234567)
 # For guidelines: https://en.opensuse.org/openSUSE:Creating_a_changes_file_(RPM)#Changelog_section_.28.25changelog.29
 
+- SUSE Manager: Universe is not required for installing spacecmd
+  on Ubuntu (bsc1187337)
 - In the Prometheus chapter of the Administration Guide advise to store
   data locally (bsc#1188855)..
```

Comment:
@keichwa I guess that part of the changelog is not right, as it contains two dots ^, make sure you fix it so Joseph doesn't need to take care of it on packaging day :-)